pandas/tests/frame/test_merge.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
```

Comment:
no, these need to go in ``pandas/tests/tools/test_merge.py``. just fit them in where appropriate.

``.join`` is fine here as its a direct / used method on DataFrame. ``.merge`` is a direct calling of the ``pd.merge`` where LOTS of tests already exist. Test location is very important to avoid future confusion.

In fact I suspect most of these are duplicated tests, pls only add as appropriate.